server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -551,4 +551,34 @@ class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
     }
     assert(e.getMessage.contains("Server returned HTTP response code: 403")) // 403 Forbidden
   }
+
+  integrationTest("gcs support") {
```

Comment:
nit: Could you use `ignore` for this test for now? We will set up the credentials and enable it later.

Reply:
I used `ignore`  instead of `integrationTest`.